### PR TITLE
use-academicons:0.1.0

### PR DESCRIPTION
**Pull request by bpkleer**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: This package provides functions to use icons of Academicons conveniently by leveraging the desktop fonts. One can simply add icons via their name once the font is installed.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

File: packages/preview/academicons/0.1.0/lib.typ [hunk 1]
```diff
@@ -0,0 +1,81 @@
+//! typst-fontawesome
+//!
+//! https://github.com/duskmoon314/typst-fontawesome
```
[the remark this answers is not in the document]
Comment (JamesxX) [reply packages/preview/academicons/0.1.0/lib.typ:3]: Also this might end up clashing with the annotation syntax that has been proposed

**Comment by bpkleer** [comment]
Thanks for the quick review and sorry for the copy/paste error. Regarding authorship, I sent an email, but I'm waiting for the answer.

**Comment by elegaanz** [comment]
Thanks for the package submission. It all looks good to me, excepted for the name that is canonical (it is true `fontawesome` was accepted, but that was a mistake). Could you please change it to make it a bit more unique?

**Comment by bpkleer** [comment]
> Thanks for the package submission. It all looks good to me, excepted for the name that is canonical (it is true `fontawesome` was accepted, but that was a mistake). Could you please change it to make it a bit more unique?

How about "using-academicons" or "import-academicons" ?

**Comment by elegaanz** [comment]
I personally feel like it would be a bit redundant with the `#import` keyword, but it would fit the naming guidelines I think, so if you want you can use that :)

**Pull request merged**